packages/blockchain-wallet-v4-frontend/src/data/components/importBtcAddress/sagas.spec.js

```diff
@@ -0,0 +1,76 @@
+import { testSaga } from 'redux-saga-test-plan'
+import { coreSagasFactory } from 'blockchain-wallet-v4/src'
+import { actions } from 'data'
+import sagas from './sagas.js'
+import Bitcoin from 'bitcoinjs-lib'
+
+const api = {}
+const networks = { btc: Bitcoin.networks['bitcoin'] }
+const coreSagas = coreSagasFactory({ api })
+const { importLegacyAddress, importBtcAddressSubmitClicked } = sagas({
+  api,
+  coreSagas,
+  networks
+})
+const logLocation = 'components/importBtcAddress/sagas'
+
+describe('importBtcAddress sagas', () => {
+  describe('importBtcAddressSubmitClicked', () => {
+    describe('non bip-38 addr', () => {
+      const saga = testSaga(importBtcAddressSubmitClicked)
+      it('should select form state', () => {
+        // Fails
+        // saga.next().select(selectors.form.getFormValues('importLegacyAddress'))
+        saga.next()
+      })
```

Comment:
Do you know why this fails @tony-blockchain

Reply:
It's likely that `selectors.form.getFormValues('importBtcAddress')` fails without form in state
general saga `select` tests don't need the state to be present